comments/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
```

Comment:
imports are supposed to follow a certain order and grouping. 
PEP8 has their "rules" for import grouping and order laid out pretty well, and Django adds to that. Here is more on that with examples

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports